src/common/object-name.js

```diff
@@ -0,0 +1,14 @@
+import { connectStore } from 'utils'
+import { createGetObject } from 'selectors'
+import React, { Component } from 'react'
+
+@connectStore(() => {
+  const object = createGetObject()
+  return (state, props) => ({object: object(state, props)})
+})
+export default class ObjectName extends Component {
+  render () {
+    const { object } = this.props
+    return <span>{object && object.name_label}</span>
+  }
+}
```

Comment:
I submit the idea because I need exactly the same component in 2 places (the other use is not maintained yet in my branch), so I wanted to avoid a dupplication. But I agree it is not so "common" and recommended to use than other common contents...